.github/workflows/build.yaml

```diff
@@ -0,0 +1,39 @@
+name: Deploy workflow for Obsrv-API
+
+on:
+  push:
+    tags:
+      - "*/*"
+
+jobs:
+  docker:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-2
+
+      - name: Authenticate Docker
+        run: |
+          docker login -u "sanketikahub" -p "${{ secrets.DOCKER_PASSWORD }}" docker.io
+
+      - name: Build Docker image
+        run: |
+          docker build -t sanketikahub/obsrv-api-service:1.0.0 -f Dockerfile .
```

Comment:
are we not changing the version of the docker image every time we built?

Reply:
Updated the config.